e2e-tests/cypress/support/guidance-pages/guidancePageNavigation.js

```diff
@@ -0,0 +1,18 @@
+module.exports = (navigationType) => {
+  let identifier;
+  switch (navigationType) {
+    case 'next':
+      identifier = '[data-cy="pagination-next"]';
+      break;
+    case 'previous':
+      identifier = '[data-cy="pagination-previous"]';
+      break;
+    case 'start':
+      identifier = '[data-cy="pagination-previous"]';
```

Comment:
Is the data-cy supposed to be the same for lines 8 and 11?

Reply:
Good catch, fixing